### ISSUE TITLE
a textsymboliser is used for charactermarkers

### ISSUE THREAD

**Issue by pvgenuchten** [GeoCat/bridge-style#21]
This results in secondary symbol levels not being drawn due to label-collision

In stead use a pointsymboliser:

```xml
<PointSymbolizer>
<Graphic>
<Mark>
  <WellKnownName>ttf://Dingbats#0x3A</WellKnownName>
  <Fill><CssParameter name="fill">#1600DB</CssParameter></Fill>
</Mark>
<Opacity>1</Opacity>
<Size>20</Size>
</Graphic>
</PointSymbolizer>
```

newer versions of geoserver also support se1.1

```xml
 <se:PointSymbolizer>
            <se:Graphic>
              <se:Mark>
                <se:OnlineResource xlink:type="simple" xlink:href="ttf://Dingbats"/>
                <se:Format>ttf</se:Format>
                <se:MarkIndex>10733</se:MarkIndex>
                <se:Fill>
                  <se:SvgParameter name="fill">#ff0000</se:SvgParameter>
                </se:Fill>
              </se:Mark>
              <se:Size>29</se:Size>
            </se:Graphic>
          </se:PointSymbolizer>
```

**Comment by volaya** [comment]
Nice suggestion. I implemented it. However, QGIS support for markers with more than a single character. For that case, I am keeping the text symbolizer

**Comment by pvgenuchten** [comment]
> However, QGIS support for markers with more than a single character. For that case, I am keeping the text symbolizer

Do you mean as layers on top of each other? That would give 2 pointsymbolisers, no?

**Comment by volaya** [comment]
No. I mean that there is a "character" value that is supposed to have a single character, like 'a' or 'b', but not something like 'hello'. QGIS supports long strings for font markers, so in that case I use a text symbolizer, not the point symbolizer

**Comment by pvgenuchten** [comment]
So QGIS facilitates to use pointsymboliser as a label engine, curious. Never seen that on other products. Mind that a conversion from point-symboliser in qgis to label in geoserver will potentially give quite unexpected results, due to label-collision detection in geoserver.
